build/windows/tests/10.0.x/build/jss-integrated-mode/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# escape=`
+
+# The purpose of this DockerFile is to install NodeJS for JSS Integrated Mode. This can be used in CM or CD
+
+ARG BASE_IMAGE
+
+FROM $BASE_IMAGE as base
+
+ARG NODEJS_VERSION
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN New-Item -Path 'C:\\inetpub\\wwwroot\\temp\\install\\setup' -ItemType 'Directory' -Force | Out-Null; `
+    & curl.exe -sS -L -o C:\\inetpub\\wwwroot\\temp\\install\\setup\\node.msi "https://nodejs.org/download/release/v$Env:NODEJS_VERSION/node-v$Env:NODEJS_VERSION-x64.msi";
+
+RUN $env:INSTALL_TEMP = 'C:\\inetpub\\wwwroot\\temp\\install'; `
```

Comment:
Can these two instructions be combined into one RUN statement?

Reply:
Done